internal/cloudsql/refresh_test.go

```diff
@@ -119,6 +121,127 @@ func TestRefreshFailsFast(t *testing.T) {
 	}
 }
 
+type tokenResp struct {
+	tok *oauth2.Token
+	err error
+}
+
+type fakeTokenSource struct {
+	responses []tokenResp
+	mu        sync.Mutex
+	i         int
```

Comment:
nit: I have no idea what `i` is for if there is also a `count`

Reply:
I'll consolidate to just use one.